tests/core/Did.spec.ts

```diff
@@ -1,8 +1,57 @@
+import Cryptography from '../../lib/core/versions/latest/util/Cryptography';
 import Did from '../../lib/core/versions/latest/Did';
+import DidPublicKeyModel from '../../lib/core/versions/latest/models/DidPublicKeyModel';
+import Encoder from '../../lib/core/versions/latest/Encoder';
+import ErrorCode from '../../lib/core/versions/latest/ErrorCode';
+import JasmineSidetreeErrorValidator from '../JasmineSidetreeErrorValidator';
+import KeyUsage from '../../lib/core/versions/latest/KeyUsage';
 
 describe('DID', async () => {
-  it('isDid() should return false if DID method is not in the DID.', async () => {
-    const isDid = Did.isDid('did:abcdefg');
-    expect(isDid).toBeFalsy();
+  describe('create()', async () => {
+    it('should create a short-form DID succssefully.', async () => {
+      const expectedDidMethodName = 'did:sidetree:';
+      const uniqueSuffix = 'abcdefg';
+      const didString = expectedDidMethodName + uniqueSuffix;
+      const did = Did.create(didString, expectedDidMethodName);
+      expect(did.didMethodName).toEqual(expectedDidMethodName);
+      expect(did.encodedDidDocument).toBeUndefined();
+      expect(did.isShortForm).toBeTruthy();
+      expect(did.shortForm).toEqual(didString);
+      expect(did.uniqueSuffix).toEqual(uniqueSuffix);
+    });
+
+    it('should throw if DID given does not match the expected DID method name.', async () => {
+      JasmineSidetreeErrorValidator.expectSidetreeErrorToBeThrown(
+        () => Did.create('did:sidetree:EiAgE-q5cRcn4JHh8ETJGKqaJv1z2OgjmN3N-APx0aAvHg', 'did:sidetree2:'),
+        ErrorCode.DidIncorrectPrefix
+      );
+    });
+
```

Comment:
Suggestion: do you think we need a test for when the DID is longform?

Reply:
Added.